sdk/storage/Azure.Storage.Common/src/Shared/StorageClientOptions.cs

```diff
@@ -67,12 +67,20 @@ public static HttpPipelinePolicy AsPolicy<TUriBuilder>(this AzureSasCredential c
         /// </summary>
         /// <param name="credential">Credential to use.</param>
         /// <param name="options"> The <see cref="ISupportsTenantIdChallenges"/> to apply to the credential. </param>
+        /// <param name="audienceScope">The scope to request for the TokenCredential.</param>
         /// <returns>An authentication policy.</returns>
-        public static HttpPipelinePolicy AsPolicy(this TokenCredential credential, ClientOptions options) =>
-            new StorageBearerTokenChallengeAuthorizationPolicy(
+        public static HttpPipelinePolicy AsPolicy(this TokenCredential credential, ClientOptions options, string audienceScope = StorageScope)
```

Comment:
Nit - consider making the default `null` since you've got the `IsNullOrEmpty` check below and then we don't have the default scope specified twice.

Reply:
Fixed.